python/nnvm/top/vision.py

```diff
@@ -0,0 +1,54 @@
+
+# pylint: disable=invalid-name, unused-argument
+"""Definition of nn ops"""
+from __future__ import absolute_import
+
+import topi
+import tvm
+from . import registry as reg
+from .registry import OpPattern
+
+@reg.register_compute("shortcut")
```

Comment:
under what namespace is the registered operator? @tqchen

Reply:
I think for now they are registered as yolo2_shortcut